### ISSUE TITLE
Load and Evaluate a saved model

### ISSUE THREAD

**Issue by Yogurt928** [Microsoft/CNTK#2499]
Hi,
I'm training a binary classifier (A feedforward network):
```python
input = C.input_variable(input_dim)
label = C.input_variable(num_output_classes)
z = create_model(input)
	
loss = C.cross_entropy_with_softmax(z, label)
eval_error = C.classification_error(z, label)
lr_schedule = learning_rate_schedule(lr_per_sample, UnitType.sample)
learner = sgd(z.parameters, lr_schedule)
trainer = Trainer(z, (loss, eval_error), [learner])
```

After training, I will test the model and also save the model:
```python
z.save(os.path.join(subModelPath, recordName + ".dnn"))
test_error=trainer.test_minibatch({input : features_test, label : labels_test})
acc=1-test_error
```
Here I can get a accuracy as 0.925625

I try to load the model and evaluate it with the same test data:
```python
z = C.load_model(os.path.join(modelPath, recordName + ".dnn"))
out = C.softmax(z)
predictions=out.eval({out.arguments[0]:features_test})
true_label = [np.argmax(label) for label in labels_test]
predict_label = [np.argmax(predictions[row,:]) for row in range(predictions.shape[0])]
	
correctNum=0
for index in range(len(predictions)):
	if true_label[index]==predict_label[index]:
		correctNum=correctNum+1
acc=correctNum/len(predict_label)
print("Accuracy: {}".format(acc))
```
But here I get an accuracy as 0.54, is there anything wrong during loading the model?
Thanks in advance.

**Comment by tangyuq** [comment]
Assuming that your test data format is the same as in the training time,  can you please try the following and see what is the outcome? 
```python
z = C.load_model(os.path.join(modelPath, recordName + ".dnn"))
label = C.input_variable(num_output_classes)
eval_error = C.classification_error(z, label)
acc = C.reduce_mean(eval_error, axis=z.dynamic_axes[0])

acc_value = acc.eval({z.arguments[0]: features_test , label: labels_test})
print("Accuracy: {}".format(acc_value))
```

**Comment by Yogurt928** [comment]
@tangyuq 
Thanks for your reply.
The code can get the same result 0.46=1-0.54
But I have found the reason, I forgot to do some process on the test data as in training time, which makes these two result looks different.  After doing the same process, I can get the same accuracy 0.925625 now.
Sorry for my careless and thanks for your advice which help me do the evaluation in a more simple way! ^_^
